PERFORMANCE

```diff
@@ -37,6 +37,55 @@ To characterize Chapel performance, generally speaking...
   assignments are used to move chunks of data between locales.
 
 
+Experimental flags for improving performance
+--------------------------------------------
+
+The following compiler flags are intended to provide a preview of
+performance improvements that we anticipate being delivered by the
+compiler automatically as it matures.  Both are available for use "at
+your own risk" in that they are not guaranteed to maintain program
+correctness (details available after each flag).
+
+* chpl -sassertNoSlicing=true ...
```

Comment:
Does that work for config params?  I thought it was consts/vars (execution-time flags) only?